frontend/src/components/tables/MarketTables.jsx

```diff
@@ -3,32 +3,25 @@ import { Link } from 'react-router-dom';
 import { API_URL } from '../../config';
 
 const TableHeader = () => (
-  <thead className='bg-gray-50'>
+  <thead className='bg-gray-900'>
     <tr>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        Trade
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        🪙
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        Question
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        📅 Closes
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        Creator
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        👤 Users
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        📊 Size
-      </th>
-      <th className='px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider'>
-        💬
-      </th>
+      {[
+        'Trade',
+        '🪙',
+        'Question',
+        '📅 Closes',
```

Comment:
Left a comment about this under, `MarketDetailsLayout.jsx` as well.

This applies to the case where a market has been RESOLVED as either YES or NO.

So basically, in both the `MarketDetailsLayout.jsx` and the `MarketTables.jsx` we should have additional logic to display whether the market has been resolved or not and the resolution direction, either YES or NO.

Reply:
Fixed here